features/term.feature

```diff
@@ -39,6 +39,12 @@ Feature: Term fixtures
       5
       """
 
+    When I run `wp db query "SELECT COUNT(1) FROM wp_term_taxonomy WHERE term_id IN ( SELECT parent FROM wp_term_taxonomy WHERE parent != 0 ) AND taxonomy = 'post_tag'"`
```

Comment:
Couldn't come up with a better way to write this test, but this query essentially says "find me terms that are parents and are tags" and the expected outcome should be 0.